readme.md

```diff
@@ -162,7 +162,9 @@ Change the text.
 
 Change the spinner color.
 
-#### .promise(action, [options|text])
+### Static
```

Comment:
Remove this and make the `ora.promise` title larger instead (`### ora.promise(...)`).